packages/ffe-account-selector-react/src/index.d.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, HTMLAttributes, ChangeEventHandler } from 'react';
+import { Account } from './util/types';
```

Comment:
Will typescript understand the Account-type defined in this file?

Reply:
No, it will not. Made an Account type for this in the index.d.ts -file